tests/src/test/scala/cats/tests/ReaderWriterStateTTests.scala

```diff
@@ -279,53 +280,77 @@ class ReaderWriterStateTTests extends CatsSuite {
       ContravariantTests[ReaderWriterStateT[ListWrapper, ?, Int, String, Int]].contravariant[String, String, String])
+    checkAll("Contravariant[ReaderWriterStateT[ListWrapper, ?, Int, String, Int]]",
+      SerializableTests.serializable(Contravariant[ReaderWriterStateT[ListWrapper, ?, Int, String, Int]]))
+
     checkAll("ReaderWriterStateT[ListWrapper, Int, Int, String, Int]",
       ProfunctorTests[ReaderWriterStateT[ListWrapper, ?, Int, String, ?]].profunctor[Int, Int, Int, Int, Int, Int])
+    checkAll("Profunctor[ReaderWriterStateT[ListWrapper, ?, Int, String, ?]]",
+      SerializableTests.serializable(Profunctor[ReaderWriterStateT[ListWrapper, ?, Int, String, ?]]))
+
     checkAll("ReaderWriterStateT[ListWrapper, Int, Int, Int, Int]",
       BifunctorTests[ReaderWriterStateT[ListWrapper, String, Int, ?, ?]].bifunctor[Int, Int, Int, Int, Int, Int])
+    checkAll("Bifunctor[ReaderWriterStateT[ListWrapper, String, Int, ?, ?]]",
+      SerializableTests.serializable(Bifunctor[ReaderWriterStateT[ListWrapper, String, Int, ?, ?]]))
   }
 
   {
     implicit val LWM: Monad[ListWrapper] = ListWrapper.monad
 
     checkAll("ReaderWriterStateT[ListWrapper, String, Int, String, Int]",
       MonadTests[ReaderWriterStateT[ListWrapper, String, Int, String, ?]].monad[Int, Int, Int])
+    checkAll("Monad[ReaderWriterStateT[ListWrapper, String, Int, String, ?]]",
+      SerializableTests.serializable(Monad[ReaderWriterStateT[ListWrapper, String, Int, String, ?]]))
   }
 
   {
     implicit val LWM: Monad[ListWrapper] = ListWrapper.monad
 
     checkAll("ReaderWriterStateT[ListWrapper, String, Int, String, Int]",
       MonadStateTests[ReaderWriterStateT[ListWrapper, String, Int, String, ?], Int].monadState[Int, Int, Int])
+    checkAll("MonadState[ReaderWriterStateT[ListWrapper, String, Int, String, ?]. Int]",
+      SerializableTests.serializable(MonadState[ReaderWriterStateT[ListWrapper, String, Int, String, ?], Int]))
   }
 
   {
     implicit val LWM: MonadCombine[ListWrapper] = ListWrapper.monadCombine
 
     checkAll("ReaderWriterStateT[ListWrapper, String, Int, String, Int]",
       MonadCombineTests[ReaderWriterStateT[ListWrapper, String, Int, String, ?]].monadCombine[Int, Int, Int])
+    checkAll("MonadCombine[ReaderWriterStateT[ListWrapper, String, Int, String, ?]]",
+      SerializableTests.serializable(MonadCombine[ReaderWriterStateT[ListWrapper, String, Int, String, ?]]))
   }
 
   {
     implicit val LWM: Monad[ListWrapper] = ListWrapper.monad
 
     checkAll("ReaderWriterStateT[ListWrapper, String, Int, String, Int]",
       MonadReaderTests[ReaderWriterStateT[ListWrapper, String, Int, String, ?], String].monadReader[String, String, String])
+
+    // check serializable using Option
+    checkAll("MonadReader[ReaderWriterStateT[Option, String, Int, String, ?], String]",
+      SerializableTests.serializable(MonadReader[ReaderWriterStateT[Option, String, Int, String, ?], String]))
```

Comment:
If I used `ListWrapper here I got

```
Message: cannot assign instance of scala.collection.immutable.List$SerializationProxy to field cats.tests.ListWrapper.list of type scala.collection.immutable.List in instance of cats.tests.ListWrapper
```

So I used `Option` like the analogous test for `Kleisli`.